viewer/packages/data-providers/src/image-360-data-providers/descriptor-providers/datamodels/cdm/Cdf360CdmDescriptorProvider.ts

```diff
@@ -0,0 +1,251 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { CogniteClient, DirectRelationReference, FileInfo } from '@cognite/sdk';
+import {
+  Historical360ImageSet,
+  Image360Descriptor,
+  Image360DescriptorProvider,
+  Image360FileDescriptor,
+  InstanceIdentifier,
+  QueryNextCursors
+} from '../../../../types';
+import { Cdf360FdmQuery, get360CollectionQuery } from './get360CdmCollectionQuery';
+import assert from 'assert';
+import { Euler, Matrix4 } from 'three';
+import { DataModelsSdk } from '../../../../DataModelsSdk';
+import chunk from 'lodash/chunk';
+import zip from 'lodash/zip';
+import groupBy from 'lodash/groupBy';
+import partition from 'lodash/partition';
```

Comment:
```suggestion
import { chunk, zip, groupBy, partition } from 'lodash';
```

Reply:
Eslint asks me to import the modules individually, so I'll thus keep it as is